src/integ.gitlab-runner-autoscaling.ts

```diff
@@ -21,8 +23,33 @@ const runner = new GitlabRunnerAutoscaling(stack, 'TestRunnerAutoscaling', {
     hostPath: '/tmp/cache',
     containerPath: '/tmp/cache',
   }],
-  minCapacity: 1,
+  minCapacity: defaultCapacity,
+  maxCapacity: defaultCapacity,
   spotInstance: true,
+  alarms: [
+    {
+      AlarmName: 'GitlabRunnerDiskUsage',
+      MetricName: 'disk_used_percent',
+      Threshold: 50,
+    },
+  ],
 });
 
+/**
+ * Scheduled scaling
+ * https://docs.aws.amazon.com/cdk/api/latest/docs/aws-autoscaling-readme.html#scheduled-scaling
+ */
+runner.autoscalingGroup.scaleOnSchedule('StopOnWeekends', {
+  schedule: asg.Schedule.cron({ weekDay: 'fri', hour: '18', minute: '0' }),
+  minCapacity: 0,
+  maxCapacity: 0,
+});
+
+runner.autoscalingGroup.scaleOnSchedule('WorkOnWeekdays', {
```

Comment:
Let's make it become to  runner's function ?!  like this:
```ts
runner.workOnWeekdays({
  schedule: { weekDay: 'fri', hour: '18', minute: '0' },
  minCapacity: defaultCapacity,
  maxCapacity: defaultCapacity,
}
```

Reply:
@guan840912 I think this should be imprelemented in another issue.

Reply:
ok LGTM